contents/docs/experiments/manual.mdx

```diff
@@ -8,136 +8,314 @@ availability:
--   You want to test which flow leads to the highest number of purchases.
--   You want to test if certain deals & promotions do better with users in a specific country.
--   You want to test if showing your website copy in German does better than English in Austria.
--   You want to test if certain copy does better within a cohort of new users.
+- How changes to your onboarding flow affect your signup rate.
+- If different designs of your app's dashboard increase user engagement and retention.
+- The impact of offering a free trial period versus a money-back guarantee to determine which approach results in more paying customers.
 
-## Creating an experiment
+A/B tests are also sometimes referred to as "experiments", and this is how we refer to them in the PostHog app.
 
-Creating an experiment starts with a hypothesis you want to test. Let's use an example of whether a visual update to the sign up button will increase clicks for that button.
+## How to create an experiment in PostHog
 
-After clicking on "new experiment" in the Experiments page, we should see a form like this:
+To create a new experiment, go to the [experiments tab](https://app.posthog.com/experiments) in the PostHog app, and click on the "New experiment" button in the top right.
 
-![Create experiment](../../images/features/experiments/create-experiment.png)
+This presents you with a form where you can complete the details of your new experiment:
 
-#### 1. Experiment name, feature flag, and testing variants
+![How to create an experiment in PostHog](../../images/features/experiments/create-experiment.png)
 
-Let's start in the upper left corner by filling out a name, description, and feature flag for our experiment. Fill in a name for your feature flag - it will be created here by default, linked to the experiment rather than existing as its own entity.
+Here's a breakdown of each field in the form:
 
-![Name, feature flag, and variants](../../images/features/experiments/create-experiment-name-variants.png)
+### Feature flag key
 
-We can also include multivariants for our testing changes. For example, instead of running our experiment with just "control" and "test", we can test for the green visual button change or the orange one or our original control.
+Each experiment is backed by a [feature flag](/docs/feature-flags/manual). In your code, you use this feature flag key to check which experiment variant the user has been assigned to. 
 
-#### 2. Participant selection and experiment goal/metric
+> **Advanced:** It's possible to create experiments without using PostHog's feature flags (for example, if you you're a different library for feature flags). For more info, read our docs on [implementing experiments without feature flags](// TODO add link here) 
 
-Next, we want to select our experiment's universe. Here, we've decided that for our sign up button experiment we want to test our hypothesis on users from Asia.
+### Experiment variants
```

Comment:
Technically, you are missing description here, but its optional. Don't know if it matters to include.